wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Command.java

```diff
@@ -287,6 +287,17 @@ public SequentialCommandGroup andThen(Command... next) {
     return group;
   }
 
+  /**
+   * Creates a new command that runs this command and the deadline in parallel, finishing when the
+   * deadline finishes.
+   *
+   * @param deadline the deadline of the command group
+   * @return the decorated command
+   */
```

Comment:
This should include the standard composition warning. (You can copy-paste from the neighboring methods).